### ISSUE TITLE
Curl 7.40.0 issue

### ISSUE THREAD

**Issue by TobiasScheinert** [pydio/pydio-core#961]
Hi,

after a update of my Solaris 11 machine, curl was updated to version 7.40.0. After that I've get following error from pydio:
```
stream_wrapper_register() [<a href='function.stream-wrapper-register'>function.stream-wrapper-register</a>]: Protocol smb:// is already defined.
```
I'm using the newest version of pydio (6.8.0). If you take a look at the curl changelog, you will find:
```
smb: Added initial support for the SMB/CIFS protocol 
```

The OwnCloud had a similar bug:
 [https://github.com/owncloud/core/issues/13485](https://github.com/owncloud/core/issues/13485)

Has anybody run into the same issue?

Greeting Tobias

**Comment by cdujeu** [comment]
Hi Tobias, 
There will be a conflict with our "smb://" protocol indeed. Not tested but seems logical.
Until it's fixed, check the access.smb plugin
> Inside file smb.php, look for the occurences of smb://, and at the very end of the file the stream_wrapper_register call, and name the protocol something else (for example "smb.custom")
> Inside file smbAccessWrapper, we translate ajxp.smb:// to smb://  : translate to smb.custom:// instead.
Charles

**Comment by cdujeu** [comment]
Fixed by a072ed3c851a95350c500114fda2bb4d620d38a7
